components/shell/bash/Makefile

```diff
@@ -73,54 +75,59 @@ LIBS =	-lc -lsocket -lgen
 LDFLAGS =	$(LD_Z_REDLOCSYM)
 
 # The bash test harness needs the GNU userland utilities
-TEST_PATH = "PATH=$(BUILD_DIR_32):$(GNUBIN):$(USRBINDIR)"
-
-CONFIGURE_ENV +=	LIBS="$(LIBS)"
-
-CONFIGURE_OPTIONS  +=		--localstatedir=/var
-CONFIGURE_OPTIONS  += 		--enable-alias
-CONFIGURE_OPTIONS  += 		--enable-arith-for-command
-CONFIGURE_OPTIONS  += 		--enable-array-variables
-CONFIGURE_OPTIONS  += 		--enable-bang-history
-CONFIGURE_OPTIONS  += 		--enable-brace-expansion
-CONFIGURE_OPTIONS  += 		--enable-casemod-attributes
-CONFIGURE_OPTIONS  += 		--enable-casemod-expansions
-CONFIGURE_OPTIONS  += 		--enable-command-timing
-CONFIGURE_OPTIONS  += 		--enable-cond-command
-CONFIGURE_OPTIONS  += 		--enable-cond-regexp
-CONFIGURE_OPTIONS  += 		--enable-coprocesses
-CONFIGURE_OPTIONS  += 		--enable-debugger
-CONFIGURE_OPTIONS  += 		--enable-directory-stack
-CONFIGURE_OPTIONS  += 		--enable-disabled-builtins
-CONFIGURE_OPTIONS  += 		--enable-dparen-arithmetic
-CONFIGURE_OPTIONS  += 		--enable-extended-glob
-CONFIGURE_OPTIONS  += 		--enable-help-builtin
-CONFIGURE_OPTIONS  += 		--enable-history
-CONFIGURE_OPTIONS  += 		--enable-job-control
-CONFIGURE_OPTIONS  += 		--enable-multibyte
-CONFIGURE_OPTIONS  += 		--enable-net-redirections
-CONFIGURE_OPTIONS  += 		--enable-process-substitution
-CONFIGURE_OPTIONS  += 		--enable-progcomp
-CONFIGURE_OPTIONS  += 		--enable-prompt-string-decoding
-CONFIGURE_OPTIONS  += 		--enable-readline
-CONFIGURE_OPTIONS  += 		--enable-restricted
-CONFIGURE_OPTIONS  += 		--enable-select
-CONFIGURE_OPTIONS  += 		--enable-separate-helpfiles
-CONFIGURE_OPTIONS  += 		--enable-single-help-strings
-CONFIGURE_OPTIONS  += 		--disable-strict-posix-default
-CONFIGURE_OPTIONS  += 		--enable-usg-echo-default=yes
-CONFIGURE_OPTIONS  += 		--enable-xpg-echo-default=yes
-CONFIGURE_OPTIONS  += 		--enable-mem-scramble
-CONFIGURE_OPTIONS  += 		--disable-profiling
-CONFIGURE_OPTIONS  += 		--enable-static-link
-CONFIGURE_OPTIONS  += 		--enable-largefile
-CONFIGURE_OPTIONS  += 		--enable-nls
-CONFIGURE_OPTIONS  += 		--with-bash-malloc=yes
-CONFIGURE_OPTIONS  += 		--with-curses
-CONFIGURE_OPTIONS  += 		--with-installed-readline=no
-CONFIGURE_OPTIONS  +=		--infodir=$(CONFIGURE_INFODIR)
-
-COMPONENT_TEST_DIR =		$(BUILD_DIR_32)
+TEST_PATH = "PATH=$(BUILD_DIR_64):$(GNUBIN):$(USRBINDIR)"
+
+COMPONENT_PREP_ACTION = ( cd $(@D); autoconf -f -i )
+
+CONFIGURE_ENV += LIBS="$(LIBS)"
+CONFIGURE_ENV += LD_OPTIONS="-lc -lm -lgen $(LIBSOCKET) $(LIBNSL) -lresolv $(LD_Z_IGNORE)"
```

Comment:
does it require -lsocket or not? I just don't see where $(LIBSOCKET) is defined. Ah, it's used from $(LIBS)...